src/node_crypto.cc

```diff
@@ -4655,11 +4665,22 @@ class PBKDF2Request : public AsyncWrap {
   }
 
   inline void release() {
-    free(pass_);
+    if (pass_ != nullptr) {
```

Comment:
What's the reason for adding 6 more lines for comparing with a nullptr?
I can see it used in other places for some reason, though.